client/lib/presales-chat/index.ts

```diff
@@ -36,7 +40,8 @@ function getGroupName( keyType: KeyType ) {
 
 export function usePresalesChat( keyType: KeyType, enabled = true, skipAvailabilityCheck = false ) {
 	const isEnglishLocale = useIsEnglishLocale();
-	const isEligibleForPresalesChat = enabled && isEnglishLocale;
+	const { canConnectToZendesk } = useChatStatus();
```

Comment:
Just a note that this added an unnecessary availability check, with the wrong (default) group to boot: https://github.com/Automattic/wp-calypso/pull/82065